### ISSUE TITLE
Python 3.9

### ISSUE THREAD

**Issue by hudcap** [pyzo/pyzo#728]
When starting a shell using Python 3.9, I get the following error:

```
Traceback (most recent call last):
  File "C:\Program Files\pyzo\source\pyzo\pyzokernel\start.py", line 156, in <module>
    __pyzo__.introspector.set_mode("thread")
  File "C:\Program Files\pyzo\source\pyzo\yoton\channels\channels_reqrep.py", line 703, in set_mode
    if not self._thread.isAlive():
AttributeError: 'ThreadForReqChannel' object has no attribute 'isAlive'
```

Google led me to [this](https://bugzilla.redhat.com/show_bug.cgi?id=1787746), and when I changed line 703 of `channels_reqrep.py` from

```
if not self._thread.isAlive():
```

to

```
if not self._thread.is_alive():
```

everything works.
I don't really understand the connection between my Python installation and the `yoton` package included in the `pyzo` installation, and I think my solution is a hack, so unless told otherwise, I'm not going to make a PR.

**Comment by hudcap** [comment]
...and now I see this was fixed in the release notes for 4.11.0
My bad

**Comment by hudcap** [comment]
duplicate of #713 
fixed in #714

**Comment by almarklein** [comment]
Thanks for taking the effort to report, anyway ;)
